kiwi/repository/yum.py

```diff
@@ -15,343 +15,14 @@
 # You should have received a copy of the GNU General Public License
 # along with kiwi.  If not, see <http://www.gnu.org/licenses/>
 #
-import os
-from six.moves.configparser import ConfigParser
-from tempfile import NamedTemporaryFile
 
 # project
-from kiwi.defaults import Defaults
-from kiwi.logger import log
-from kiwi.repository.base import RepositoryBase
-from kiwi.utils.rpm_database import RpmDataBase
-from kiwi.path import Path
+from kiwi.repository.dnf import RepositoryDnf
 
 
-class RepositoryYum(RepositoryBase):
+class RepositoryYum(RepositoryDnf):
```

Comment:
I wonder if instead of inheritance you want to use assignment here:

```python
RepositoryYum = RepositoryDnf
```

Reply:
I'm not sure if the semantics of that are exactly what we would expect, so I went with the conservative approach and changed it to be a subclass.